samples/keys/random_hw_unique_key/sample.yaml

```diff
@@ -2,7 +2,7 @@ sample:
   description: Generate and write random keys to the KMU.
   name: Random HW Unique Key
 common:
-    tags: hw_unique_key
+    tags: random_hw_unique_key
```

Comment:
Maybe leave both of them. The tag can be used to run all HUK samples with Twister.

Reply:
Concerning tags: I recently pushed some changes to tags: https://github.com/nrfconnect/sdk-nrf/blob/main/tests/crypto/testcase.yaml

I'm starting to use a scheme where I use a common tag for things which should be able to be grouped in CI (here: `legacy`).

Maybe use something like `huk` as a tag if you need to do this as well?
There's always `--testcase-root samples/keys` in order to include both without tags as well.

Maybe @PerMac  has thoughts on best practice here

Reply:
We don't have any defined schema for tags nor any list of used tags. Maybe at one point we will do yamls evaluation and propose some schema, but right now there are no best practices, sorry.